Helper/Entity/PageHelper.php

```diff
@@ -196,7 +196,14 @@ private function strip($s, $completeRemoveTags = [])
         if ($completeRemoveTags && $completeRemoveTags !== [] && $s) {
             $dom = new \DOMDocument();
             libxml_use_internal_errors(true);
-            $dom->loadHTML(mb_convert_encoding($s, 'HTML-ENTITIES', 'UTF-8'));
+            $encodedStr = mb_encode_numericentity(
```

Comment:
This seems like it should work just fine - but I am wondering why we are encoding HTML entities and then decoding? 

Could this be more simply written as the following?

```
$encodedStr = mb_encode_numericentity($s, [0x80, 0x10fffff, 0, ~0]);
```

Unless of course a purpose is served by the back and forth conversion that is not obvious here.